test/03LessonThreeTest.js

```diff
@@ -5,33 +5,46 @@ var assert = require('assert');
 
 function buildArray(first, second, third) {
     // return array with the items first, second, and third in an array.
+    return [first, second, third];
 
 }
 
 function returnThirdItem(arr) {
     // should return the third item in the array
+    //var arr = ["a", "b", "c"];
+    return arr[2];
 
 }
 
 function setFirstItem(arr, newFirstItem) {
     // should set the first item in the array with newFirstItem
-
+    arr[0] =newFirstItem;
+    return arr;
 }
 
 function returnCenterItem(fiveByFiveArray) {
     // returns the "center" item in a 5 x 5 array
     // ex. 3 x 3 array [[1, 2, 3], [4, 5, 6], [7, 8, 9]] the center item is 5
-
+    var arr = [
+        [1, 2, 1, 4, 5],
+        [1, 2, 2, 4, 5],
+        [1, 2, 3, 4, 5],
+        [1, 2, 4, 4, 5],
+        [1, 2, 5, 4, 5]
+    ];
+    return arr[2] [2];
```

Comment:
What's the parameter of this function? fiveByFiveArray correct?

The requirement of this function is to work with the parameter given, not to create a new array.